core/builder/src/main/java/io/quarkus/builder/BuildStepBuilder.java

```diff
@@ -15,6 +15,11 @@
  * a destructor for items it produces, which will be run (in indeterminate order) at the end of processing.
  */
 public final class BuildStepBuilder {
+    private static final boolean LOG_CONFLICT_CAUSING = Boolean
+            .parseBoolean(System.getProperty("quarkus.builder.log-conflict-cause", "false"));
```

Comment:
```suggestion
    private static final boolean LOG_CONFLICT_CAUSING = Boolean
            .getBoolean("quarkus.builder.log-conflict-cause");
```

Reply:
Did not know this one existed, thank you!